### PR TITLE
LUN-262: Fix header of qrcode scanner 

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

AddCustomDiscipline --> Open QR-Code Scanner

### CODE REVIEW

File: release-notes/unreleased/LUN-262-fix-header-of-qrcode-scanner.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-262
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Fehler bei der Darstellung der Kopfzeile des QR-Codelesers wurden behoben.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/LUN-262-fix-header-of-qrcode-scanner.yml:6]: Release Note not needed, as this wasn't published. please remove

**Review by SteffiStuffel — COMMENTED** [review]
Still too big for me on Androi

d

**Comment by f1sh1918** [comment]
> Still too big for me on Androi  d

there was a random margin set :/
Pls check again

**Review by Taggotty — APPROVED** [review]
Tested on Android Emulator

**Review by SteffiStuffel — APPROVED** [review]
Tested again, looks good now. :tada:

**Pull request merged**